app/controllers/circles_controller.rb

```diff
@@ -23,6 +24,47 @@ def create
     end
   end
 
+  def edit
+    unless user_is_creator?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.